packages/client/src/pages/Game/Game.tsx

```diff
@@ -1,11 +1,38 @@
-import { Canvas } from '../../components/Canvas'
+import React, { useEffect, useRef } from 'react'
 import { Layout } from '../../components/Layout'
-import React from 'react'
+import MainShip from '../../model/MainShip'
 
 export const Game = () => {
+  const boardIndent = 32
+  const boardHeight = window.innerHeight - boardIndent
+  const boardWidth = window.innerWidth - boardIndent
+  const canvasRef = useRef<HTMLCanvasElement | null>(null)
+  const mainShip = new MainShip(boardWidth, boardHeight)
+
+  useEffect(() => {
+    animate()
+  })
```

Comment:
https://github.com/imozi/Guardian-of-the-Galaxy/pull/18#discussion_r1046816349

Reply:
done